wbstack/test/server.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
-if( $_SERVER['SERVER_NAME'] !== 'localhost' ){
-    echo 'This is only meant for testing from localhost';
+
+if( $_SERVER['SERVER_NAME'] !== 'localhost' && $_SERVER['SERVER_NAME'] !== 'api.svc'  ){
```

Comment:
Probably better to go the approach of sending the header to the service from oauth.
Right now this oauth request probably does go over the public domain in production
However this will not work for us in dev land, so probably want to also send it internally in production land (meaning we need the header)

Reply:
What header am i supposed to send?

Reply:
It should be a `HOST` header.
You can find an example calling mediawiki from some other wbstack code at https://github.com/wbstack/api/blob/c091b3c2a66766b1762024c9f74c6278df888c7c/app/Jobs/MediawikiInit.php#L42

Reply:
I think I see this done in a later commit now :D